src/docs/devices/DETA-Grid-Connect-Single-Gang-Two-Way-6951HA/index.md

```diff
@@ -74,136 +65,87 @@ To gain individual control of button LEDs, remove specific diodes and solder a w
 
 ## Configuration Examples
 
-### 1 Gang Configuration for the BK72XX device
+### Series 1 (WB3S) Configuration Examples
```

Comment:
This is basically half of the 6952HA config (for one light switch instead of two). Only notable exception is that `activation_status_1` becomes `P8` instead of `P7`.